migrations/826-user-drop-source.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users DROP COLUMN `source`;
```

Comment:
There are quite a few users that have the `source` coming from from marketplace:
```
mysql> select source, count(*) from users group by source;
+--------+----------+
| source | count(*) |
+--------+----------+
|      0 |  4500760 |
|      1 |    20194 |
|      2 |    89662 |
+--------+----------+
3 rows in set (2,19 sec)
```

For reference, the source values: https://github.com/mozilla/olympia/pull/669/files#diff-0383f36a36f392e46d9070ecdc5184a3L542
- 0: unknown source (all of the AMO users I believe)
- 1: users from the marketplace (without browserid)
- 2: users from the marketplace (with browserid)
- 3: users from amo (with browserid): there's none of those